File: modul6/C/PRAK601_2310817210017_MUHAMMADNURHUDABAIHAQI.c
#include <stdio.h>

int main() {
    int baris, kolom, a, b;
    scanf("%d %d", &baris, &kolom);
    
    int matrix[baris][kolom];
    for (a = 0; a < baris; a++) {
        for (b = 0; b < kolom; b++) {
            scanf("%d", &matrix[a][b]);
        }
    }

    for (a = 0; a < baris; a++) {
        for (b = 0; b < kolom; b++) {
            printf("%d ", matrix[a][b]);
        }
        printf("\n");
    }

    return 0;
}

File: modul6/PYTHON/PRAK601_2310817210017_MUHAMMADNURHUDABAIHAQI.py
baris, kolom = map(int, input().split())
input_matriks = list(map(int, input().split()))
matrix = [input_matriks[a * kolom:(a + 1) * kolom] for a in range(baris)]

for a in range(baris):
    for b in range(kolom):
        print(matrix[a][b], end=" ")
    print()
